src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/SearchPageTelemetryEvent.cs

```diff
@@ -18,12 +21,16 @@ public class SearchPageTelemetryEvent : TelemetryEvent
             int pageIndex,
             int resultCount,
             TimeSpan duration,
+            IEnumerable<TimeSpan> sourceTimings,
+            TimeSpan aggregationTime,
             LoadingStatus loadingStatus) : base("SearchPage")
         {
             base["ParentId"] = parentId.ToString();
             base["PageIndex"] = pageIndex;
             base["ResultCount"] = resultCount;
             base["Duration"] = duration.TotalSeconds;
+            base["SourceTimingDuration"] = JsonConvert.SerializeObject(sourceTimings.Select(e => e.TotalSeconds)));
```

Comment:
Extra closing parenthesis at the end
```c#
            base["SourceTimingDuration"] = JsonConvert.SerializeObject(sourceTimings.Select(e => e.TotalSeconds));
```

Reply:
Yeah, the build failed. I had a local change that I had no pushed.